app/backend/approaches/approach.py

```diff
@@ -317,7 +321,8 @@ class ExtraArgs(TypedDict, total=False):
             **dimensions_args,
         )
         query_vector = embedding.data[0].embedding
-        return VectorizedQuery(vector=query_vector, k_nearest_neighbors=50, fields="embedding")
+        # TODO: use optimizations from rag time journey 3
```

Comment:
No changes are made to the actual search request, right? 
We didn't make any here:
https://github.com/microsoft/rag-time/blob/main/Journey%203%20-%20Optimize%20your%20Vector%20Index%20for%20Scale/sample/3-Vector-Compression.ipynb

Reply:
There are parameters on the search request - but we don't need to set oversampling